PLib/UI/PUIUtils.cs

```diff
@@ -74,39 +74,101 @@ public static class PUIUtils {
 		/// <param name="uiPrefab">The UI prefab to use. If null is passed, the UI should
 		/// be created and added to the GameObject hosting the controller object in its
 		/// OnPrefabInit function.</param>
-		public static void AddSideScreenContent<T>(GameObject uiPrefab = null)
+		/// <param name="inOrder">If the insertion should be performed in some order. The default of
+		/// false will simply insert at the end of the list which will make this screen appear
+		/// at the top of every side screen it applies to. Set to true to perform insertion
+		/// elsewhere in the list.</param>
+		/// <param name="insertBefore">If the insertion should be done before "insertionName" or
+		/// after "insertionName". True means before, false means after.</param>
+		/// <param name="insertionName">The name of the side screen that you want to choose
+		/// the ordering of this one around. Should be from this list:
+		/// Telepad Side Screen
+		/// Assignable Side Screen
+		/// Valve Side Screen
+		/// Tree Filterable Side Screen
+		/// Single Entity Receptacle Screen
+		/// Planter Side Screen
+		/// SingleSliderScreen
+		/// DualSliderScreen
+		/// Timed Switch Side Screen
+		/// Threshold Switch Side Screen
+		/// Research Side Screen
+		/// Filter Side Screen
+		/// Access Control Side Screen
+		/// ActivationRangeSideScreen
+		/// Gene Shuffler Side Screen
+		/// Sealed Door Side Screen
+		/// Capacity Control Side Screen
+		/// Door Toggle Side Screen
+		/// Suit Locker Side Screen
+		/// Lure Side Screen
+		/// Role Station Side Screen
+		/// Automatable Side Screen
+		/// SingleButtonSideScreen
+		/// IncubatorSideScreen
+		/// IntSliderSideScreen
+		/// SingleCheckboxSideScreen
+		/// CommandModuleSideScreen
+		/// CometDetectorSideScreen
+		/// TelescopeSideScreen
+		/// ComplexFabricatorSideScreen
+		/// MinionSideScreen
+		/// MonumentSideScreen
+		/// Logic Filter Side Screen</param>
+		public static void AddSideScreenContent<T>(GameObject uiPrefab = null, bool inOrder = false, bool insertBefore = true, string insertionName = null)
```

Comment:
This signature change is not binary compatible with the existing `AddSideScreenContent<T>(GameObject)` in PLib versions 2.23 and lower. Note that optional parameters are technically mandatory, but the _compiler_ will supply them at compile time.

Instead, consider creating an overload of `AddSideScreenContent` with your new parameters first (no need for `inOrder`), and `uiPrefab` last. Then the current method can be changed to call the new one with a `null` `insertionName`.